GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -100,6 +103,14 @@ public static void Report(Exception exception, bool isTerminating)
                 LogError(exception, isTerminating);
             }
 
+            if (exception is ExternalOperationException externalOperationException
+                && externalOperationException.InnerException is { }
+                && externalOperationException.InnerException.Message.Contains(_dubiousOwnershipSecurityConfigString))
```

Comment:
```suggestion
                && externalOperationException.InnerException?.Message?.Contains(_dubiousOwnershipSecurityConfigString)) is true
```
(`is not null` is much clearer than `is {}` - not needed here anyway.)